packages/terra-layout/themeable-variables.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
This file needs removed

Reply:
Do we know what causes this, I've seen the themable-variables.json generated in a few different places than expected.

Reply:
Yes. I fixed this issue [here](https://github.com/cerner/terra-site/pull/15/files#diff-ce225c4e66755af34002ffacbbe1cc4bR40) in the themeable-variables plugin in terra-site